### ISSUE TITLE
Bug on start

### ISSUE THREAD

**Issue by slaffko1** [avocode/avocode#1613]
* **Version**: 2.15.0

* **Web/Desktop**: Desktop

* **Platform**: win32

* **Avocode UID**: 126645
---

```
Error: Cannot call function 'setProgressBar' on missing remote object 7
Error: Cannot call function 'setProgressBar' on missing remote object 7
    at throwRPCError (C:\Users\Slaffko\AppData\Local\avocode\app-2.15.0\resources\electron.asar\browser\rpc-server.js:143:17)
    at EventEmitter.<anonymous> (C:\Users\Slaffko\AppData\Local\avocode\app-2.15.0\resources\electron.asar\browser\rpc-server.js:354:7)
    at emitMany (events.js:127:13)
    at EventEmitter.emit (events.js:204:7)
    at WebContents.<anonymous> (C:\Users\Slaffko\AppData\Local\avocode\app-2.15.0\resources\electron.asar\browser\api\web-contents.js:256:13)
    at emitTwo (events.js:106:13)
    at WebContents.emit (events.js:194:7)
    at metaToValue (C:\Users\Slaffko\AppData\Local\avocode\app-2.15.0\resources\electron.asar\renderer\api\remote.js:234:13)
    at Object.remoteMemberFunction (C:\Users\Slaffko\AppData\Local\avocode\app-2.15.0\resources\electron.asar\renderer\api\remote.js:118:18)
    at t._handleUploadStoreChang
```

**Comment by jakubzitny** [comment]
Thanks for the report @slaffko1, this is a bug, we'll try to fix it as soon as possible. Is it happening during all uploads? Can you upload anything to Avocode at all? Or did it just happen this one time?

**Comment by slaffko1** [comment]
It was one time at the start avocode. I did not upload anything

ср, 13 сент. 2017 г. в 11:07, Jakub Žitný <notifications@github.com>:

**Comment by jakubzitny** [comment]
Hi @slaffko1, we've just released Avocode 2.15.2 and this should be fixed there. Please let us know if it happens again or if you have any other problems 😉
